standards/core/incentivization.md

```diff
@@ -15,8 +15,8 @@ In an incentivized request-response protocol, only eligible (e.g., paying) clien
 Clients include eligibility proofs in their requests.
 
 Eligibility proofs are designed to be used in multiple Waku protocols, such as Store, Lightpush, and Filter.
-Store is planned to become the first Waku protocol to support incentivization.
-We discuss the proof-of-concept implementation of incentivization for Store in a later section.
+Lightpush is planned to become the first Waku protocol to support incentivization.
```

Comment:
I think it's important to note that it's _attaching RLN proofs_ to lightpush requests that's the incentivised service here. It's perfectly possible for a client to generate and attach its own proofs and simply require the service to publish on its behalf, which is out of scope for now.

Reply:
Good point, I'll clarify that.